Update 20.treatments.md (#75)
greenelab/covid19-review: content/20.treatments.md

### Treatment of Symptoms
The clinical picture of SARS-CoV-2 infection ranges from being asymptomatic, going through mild symptoms, such as cough, sore throat and fever, to severe pneumonia with respiratory failure which can lead to death. The development and the approval of a vaccine against this new microbe will require at least a year. Afterwards, the vaccine will face another challenge. It will in fact need to be produced in vast quantities in order to be distributed to all countries, while many companies do not have such production capacity. While at the moment the most effective way to fight Corona virus is social distancing, scientist are also focusing on other ways to cure the infection. Currently, treatments are going in two different directions. On one side, efforts are focusing on the development of new drugs which can cure the symptoms. On the other side, scientist are testing new antivirals that can inhibit the virus amplification in human bodies. In this section we will describe and analyse the most recent developed drugs that are undergoing new trials, as well as  to face the Corona virus disease. 

Tocilizumab 

> Background: A recent study carried out on a sample of 191 Covid-19 patients from Wuhan shows that the blood of patients who faced death contained about 74% more interleukin-6 (IL-6) compared to the survivors (Zhou et al. 2020). This indicates an inflammatory response out of control or what is called “cytokine storm”. In this context IL-6 plays a key role (Tisoncik et al. 2012). IL-6 is a pro-inflammatory cytokine belonging to the family of interleukins, immune system regulators, primarily responsible for immune cell differentiation and activation. In a healthy situation the lung respiratory epithelium together with alveolar macrophages limits the activation of the immune system as well as the alveolar spaces, ensuring homeostasis. It has been already shown that high level of IL-6, during the infection of other Corona viruses (influenza, SARS COV), lead to the high concentration of other cytokines (TFN, IL-1𝛽) with the results of heavily damaging the endotelium of blood vessels and disrupting the balance between pro and anti-inflammatory response (Park et al. 2001). The holes generated allow not just for the passage of neutrophils, macrophages and lymphocytes to the site of the infection but also the accumulation of liquids into the lungs, which is the ultimate cause of the death as per Acute Distress Respiratory Syndrome (ADRS) or Severe Acute Respiratory Syndrome (SARS) (Channappanavar and Perlman 2017), also caused by the new Corona virus . 

Reviewer: Can you please add newlines (with the "Enter" key) after each sentence?  That makes it easier to give feedback on the contribution and track changes.  This paragraph would look like:
```suggestion
Background: A recent study carried out on a sample of 191 Covid-19 patients from Wuhan shows that the blood of patients who faced death contained about 74% more interleukin-6 (IL-6) compared to the survivors (Zhou et al. 2020).
This indicates an inflammatory response out of control or what is called “cytokine storm”.
In this context IL-6 plays a key role (Tisoncik et al. 2012).
IL-6 is a pro-inflammatory cytokine belonging to the family of interleukins, immune system regulators, primarily responsible for immune cell differentiation and activation.
In a healthy situation the lung respiratory epithelium together with alveolar macrophages limits the activation of the immune system as well as the alveolar spaces, ensuring homeostasis.
It has been already shown that high level of IL-6, during the infection of other Corona viruses (influenza, SARS COV), lead to the high concentration of other cytokines (TFN, IL-1𝛽) with the results of heavily damaging the endotelium of blood vessels and disrupting the balance between pro and anti-inflammatory response (Park et al. 2001).
The holes generated allow not just for the passage of neutrophils, macrophages and lymphocytes to the site of the infection but also the accumulation of liquids into the lungs, which is the ultimate cause of the death as per Acute Distress Respiratory Syndrome (ADRS) or Severe Acute Respiratory Syndrome (SARS) (Channappanavar and Perlman 2017), also caused by the new Corona virus . 
```
You can click the button here to "Commit suggestion" to update this paragraph and then edit the others yourself.

For references, citing with the DOI is the most reliable option in Manubt.  See https://github.com/greenelab/covid19-review/pull/68#discussion_r398217044 for an example of how to get the DOI for a journal publication and cite it in the text.

Reviewer 2: Hi @aadattoli , I added the line breaks (enters) between each sentence to make it easier for people to give you feedback. 
Like @agitter said, if you could switch your citations to the manubot format, it would be super helpful! For example, for Channappanavar & Perlman, 2017, instead it would be:
`[@doi:10.1007/s00281-017-0629-x]`
If you need any help figuring this out, please ask on the gitter chat and we can try to help!

Specifically, Tocilizumab is a humanised monoclonal antibody that binds to the receptor of IL-6, de facto inhibiting the IL-6 immune cascade. 
Chinese doctors have started a trial enrolling 188 patients o which 14 with severe lung disease have shown clear sings of improvements, according to their results. 
Also, The AIFA (the Italian Drug Agency) approved the start of a new trial on March 19 recruiting patients at the initial stage of the infection. 
Together with these independent trials, Roche, also in collaboration with the FDA, will start a randomised, double-blind, placebo-controlled phase III trial early April. 
The trial will enroll 330 patients globally, which will be followed for 60 days upon use of the drug via injection to analyse its efficiency/safety (Biopharma-reporter.com).

Antivirals

1)	Interferone- β. 

Background: Interferons (IFNs) are a family of cytokines crucial to activate the first (innate) immune system response against viral infections. 
Interferons are classified into three categories based on their receptor specificity: type I, II and III (Tisoncik et al. 2012). 

> Specifically, IFNs I (IFN-𝛼 and 𝛽) and II (IFN-𝛾 ) induce the expression of antiviral proteins which bring the viral RNA to degradation (Spiegel et al. 2004). 

Reviewer 3: I think type III interferons merit additional discussion here. Type III interferons are major contributors to antiviral responses at epithelial barriers like the lung.

This paper demonstrates that IFN-lambda restricts viral replication in the lung and GI tract of SARS-CoV infected animals. IFN-lambda receptor 1 knockout animals (Ifnlr1-/-; aka Il28ra-/-) had increased viral load in affected tissues compared with WT animals. Magnitude of effect was similar to interferon-alpha receptor (the type I IFN receptor) knockouts. Interestingly, it seems that both interferons contribute as double receptor knockouts had higher viral loads than either of the single knockouts... 
[](https://jvi.asm.org/content/84/11/5670#sec-11)

So why should we care about lambda interferons? First, Eiger biopharma just announced they will be testing pegylated-IFN lambda as a treatment for COVID-19. Second, there is quite a lot of literature to suggest that IFN lambda induces antiviral responses without inducing tissue inflammation/damage like the type I IFNs... I worked on lambda interferons for several years so I am happy to write more about this!

Will add more to the tocilizumab section as I find some free time. Is anyone writing about hydroxychloroquine? Coming from a rheum background I have lots of thoughts on HCQ and some of the original papers that came out...

Reviewer 2: I can open an issue for hydroxychloroquine (and tag you), it's definitely important that we include it!

Reviewer 3: I'm happy to write a quick paragraph on it.

Author: Thank you very much all for your suggestions. Will incorporate them all ASAP. I added a paragraph on Remdesivir in the meantime, I can collaborate on the paragraph on hydroxychloroquine as well.

However, previous studies on RA showed that in patients treated with TCZ the rate of incident infections in clinical practice patients was higher than the one observed during clinical trial (doi.org/10.1093/rheumatology/ker223). 
Also, RA patients with chronic hepatitis B (HB) infection showed high risk of HB virus reactivation upon TCZ administration in combination with other RA drugs (doi.org/10.1111/1756-185X.13010). 
These last findings highlight the need to search for a balance between impairing a harmful immune response, such as the one generated by the cytokine storm, and preventing the worsening of the clinical picture of the patients by potential new viral infections. 
This aspect is probably crucial to be investigated further in the trials that are about to start. 
Perhaps, the TCZ treatment would best suit patients with severely compromised lungs due to the Covid-19 infection and are therefore at greater risk of death, in order to stop the uncontrolled immune response before it’s too late. 

Antivirals

Antivirals are an emerging category of drugs.

> Unlike antibiotics, they do not kill viruses.

Reviewer 2: ```suggestion
Unlike most antibiotics which aim to kill bacteria, antivirals exert their function by inhibiting viral proliferation in the host. 
Their mechanisms of action include the inhibition of viral protein, viral entry xyz...
```

> Rather, they inhibit the proliferation of a virus.

Reviewer 2: ```suggestion
Rather, they inhibit the proliferation of a virus.
```
```suggestion

```

> Categories may include therapies that inhibit viral proteins, inhibit viral entry, and more

Reviewer 2: ```suggestion
```

It will in fact need to be produced in vast quantities in order to be distributed to all countries, while many companies do not have such production capacity. 
While at the moment the most effective way to fight Corona virus is social distancing, scientist are also focusing on other ways to cure the infection. 
Currently, treatments are going in two different directions.
On one side, efforts are focusing on the development of new drugs which can cure the symptoms.
On the other side, scientist are testing new antivirals that can inhibit the virus amplification in human bodies. 
In this section we will describe and analyse the most recent developed drugs that are undergoing new trials, as well as  to face the Corona virus disease. 

#### Tocilizumab

A recent study carried out on a sample of 191 Covid-19 patients from Wuhan shows that the blood of patients who faced death contained about 74% more interleukin-6 (IL-6) compared to the survivors (doi.org/10.1016/s0140-6736(20)30566-3).
This indicates an inflammatory response out of control or what is called “cytokine storm”.
In this context IL-6 plays a key role (doi.org/10.1128/mmbr.05015-11).

> IL-6 is a pro-inflammatory cytokine belonging to the family of interleukins, immune system regulators, primarily responsible for immune cell differentiation and activation. Interleukins can regulate both pro- and anti-inflammatory response. In this context, severe lung infection reflects the over-production of proinflammatory interleukins. 

Reviewer 4: ```suggestion
IL-6 is a pro-inflammatory cytokine belonging to the family of interleukins, immune system regulators, primarily responsible for immune cell differentiation and activation. Interleukins can regulate both pro- and anti-inflammatory response.
In this context, severe lung infection reflects the over-production of proinflammatory interleukins. 
```

In a healthy situation the lung respiratory epithelium together with alveolar macrophages limits the activation of the immune system as well as the alveolar spaces, ensuring homeostasis.

> It has been already shown that high level of IL-6, during the infection of other Corona viruses (influenza, SARS COV), lead to the high concentration of other cytokines, such as Tumor Necrosis Factor (TNF) and IL-1𝛽. These cytokines enhance the pro-inflammatory reaction by increase acute-phase signaling, trafficking of immune cells to the site of primary infection, epithelial cell activation, and secondary cytokine production. The acutephase response to infection results in the heavily damage of the endotelium of blood vessels, which disrupts the balance between pro and anti-inflammatory response (doi.org/10.1164/ajrccm.164.10.2104013).

Reviewer 4: ```suggestion
It has been already shown that high level of IL-6, during the infection of other Corona viruses (influenza, SARS COV), lead to the high concentration of other cytokines, such as Tumor Necrosis Factor (TNF) and IL-1𝛽. These cytokines enhance the pro-inflammatory reaction by increase acute-phase signaling, trafficking of immune cells to the site of primary infection, epithelial cell activation, and secondary cytokine production.
The acute phase response to infection results in the heavily damage of the endotelium of blood vessels, which disrupts the balance between pro and anti-inflammatory response (doi.org/10.1164/ajrccm.164.10.2104013).
```

> Thus, the holes generated allow not just for the passage of neutrophils, macrophages and lymphocytes to the site of the infection but also the accumulation of liquids into the lungs, which is the ultimate cause of the death as per Acute Distress Respiratory Syndrome (ADRS) or Severe Acute Respiratory Syndrome (SARS) (doi.org/10.1007/s00281-017-0629-x), also caused by the new Corona virus . 

Reviewer 4: ```suggestion
Thus, the holes generated allow not just for the passage of neutrophils, macrophages and lymphocytes to the site of the infection but also the accumulation of liquids into the lungs, which is the ultimate cause of the death as per Acute Distress Respiratory Syndrome (ADRS) or Severe Acute Respiratory Syndrome (SARS) (doi.org/10.1007/s00281-017-0629-x), also caused by the new coronavirus. 
```

> Recently Chinese and Italian doctors have found that the the Tocilizumab (actemra, by Roche), a drug commonly used in the rheumatoid arthritis, is very effective in the cure of Covid-19 patients. 

Reviewer 4: ```suggestion
Recently Chinese and Italian doctors have found that the the Tocilizumab (actemra, by Roche), a drug commonly used in the rheumatoid arthritis, may palliate the most severe symptoms associated with COVID-19. 
```

##### Anticipated Mechanism 

Specifically, Tocilizumab is a humanised monoclonal antibody that binds to the receptor of IL-6, de facto inhibiting the IL-6 immune cascade. 

##### Current Evidence

> Chinese doctors have started a trial enrolling 188 patients o which 14 with severe lung disease have shown clear sings of improvements, according to their results. 

Reviewer 4: ```suggestion
Chinese doctors have started a trial enrolling 188 patients of which 14 with severe lung disease have shown clear sings of improvements, according to their results. 
```

However, previous studies on RA showed that in patients treated with TCZ the rate of incident infections in clinical practice patients was higher than the one observed during clinical trial (doi.org/10.1093/rheumatology/ker223). 
Also, RA patients with chronic hepatitis B (HB) infection showed high risk of HB virus reactivation upon TCZ administration in combination with other RA drugs (doi.org/10.1111/1756-185X.13010). 
These last findings highlight the need to search for a balance between impairing a harmful immune response, such as the one generated by the cytokine storm, and preventing the worsening of the clinical picture of the patients by potential new viral infections. 
This aspect is probably crucial to be investigated further in the trials that are about to start. 
Perhaps, the TCZ treatment would best suit patients with severely compromised lungs due to the Covid-19 infection and are therefore at greater risk of death, in order to stop the uncontrolled immune response before it’s too late. 

##### Summary

Summarize the state of the symptom management approach.

### Small Molecule Drugs for COVID-19

> The replication cycle of a virus within an epithelial host cell includes 6 basic steps which can be shortly summarised as follow: i) attachment of the virus to the host cell; ii) penetration by endocytosis; iii) uncoating, classically defined as the release of viral contents into the host cell; iv) biosynthesis, during which the viral genetic material enters the nucleus where it gets replicated; v) assembly, where viral proteins are translated and new viral particles are assembled; vi) release, when the new viruses are released into the extracellular environment (ISBN-13: 978-1405136457). 

Reviewer 4: ```suggestion
The replication cycle of a virus within an epithelial host cell includes 6 basic steps which can be shortly summarised as follow: i) attachment of the virus to the host cell; ii) penetration by endocytosis; iii) uncoating, classically defined as the release of viral contents into the host cell; iv) biosynthesis, during which the viral genetic material enters the nucleus where it gets replicated; v) assembly, where viral proteins are translated and new viral particles are assembled; vi) release, when the new viruses are released into the extracellular environment [@isbn:978-1405136457]. 
```

Antiviral drugs do not kill the virus, rather they inhibit its amplification by impairing one of these steps. 
Nowadays, many of these drugs act during the biosynthesis step in order to inhibit the virus’ genetic material replication. Importantly, Corona viruses, including Covid-19, are RNA viruses. 

> Differently from a DNA virus, which can use the host enzymes to propagate itself, RNA viruses depends on their own polymerase, the RNA-dependent RNA polymerase (RdRP), in order to be replicated (doi.org/10.1007/978-1-4939-2438-7; doi.org/10.1002/jmv.25761). 

Reviewer 4: ```suggestion
Differently from a DNA virus, which can use the host enzymes to propagate itself, RNA viruses depends on their own polymerase, the RNA-dependent RNA polymerase (RdRP), in order to be replicated [@doi:10.1007/978-1-4939-2438-7; @doi:10.1002/jmv.25761].
```

#### Nucleotide Analogs

Why one might use nucleotide analogs.

##### Avigan

Avigan (Flavipiravir, from Toyama chemical Fujifilm) is a drug which has been found effective to block viral amplification in the Influenza virus infection. 
Specifically, Avigan is a nucleoside precursor efficiently recognised as guanosine and adenosine analogue by the virus polymerase (RNA-dependent RNA polymerase). 

> While a single incorporation does not influence RNA transcription, multiple events of incorporation lead to the arrest of RNA synthesis (doi.org/10.1371/journal.pone.0068347). Importantly It was already shown that Avigan is able to give 100% coverage against the Ebola virus in mice (doi.org/10.1038/s41422-020-0282-0). 

Reviewer 4: ```suggestion
While a single incorporation does not influence RNA transcription, multiple events of incorporation lead to the arrest of RNA synthesis (doi.org/10.1371/journal.pone.0068347). Importantly It was already shown that Avigan is able to give 100% coverage against the Ebola virus in mice [@doi:10.1038/s41422-020-0282-0]. 
```

Furthermore, there are evidences that drug might also work against Corona virus infection. 
For instance, a recent study showed its effectiveness compared to other antivirals such as lopinavir and ritonavir. 
The drug was tested on a sample of 80 patients (35 experimental sample, 45 control group) and increased the speed of recovery (measured as viral clearance from the patient by RT-PCR) of about 4 days, compared to the control sample treated with the other anti-virals, such as Lopinavir and Ritonavir. 

> Also lung body scan seem to look better in about 91% of the patients analysed (doi.org/10.1016/j.eng.2020.03.007). 

Reviewer 4: ```suggestion
Also lung body scan seem to look better in about 91% of the patients analysed [@doi:10.1016/j.eng.2020.03.007]. 
```

However, the size of the sample is too small to give a powerful statistics, as well as the choice of the patients did not take into consideration important factors such as previous clinical conditions, sex, while there was no age categorisation. 

###### Current Evidence

A list of current studies and their results, using carefully the information requested in the therapeutic paper tickets.

###### Summary

Summarize the state of the antiviral approach.

##### Remdesivir

> Remdesivir (Gilead) is an adenosine analogue which once incorporated into the nascent viral chain causes premature termination of transcription (doi.org/10.1038/s41422-020-0282-0).

Reviewer 4: ```suggestion
Remdesivir (Gilead) is an adenosine analogue which once incorporated into the nascent viral chain causes premature termination of transcription [@doi:10.1038/s41422-020-0282-0].
```

Remdesivir acts at early stage of infection. 

> Previous works already showed that Remdesivir was effective in the treatment of Covid-19 similar viral pathogens, such as SARS-Cov and MERS-Cov, in animal models, while a recent study showed that Remdesivir resulted in 100% protection against the Ebola virus in human liver Cancer cell line, which is also sensitive to the novel Corona virus (doi.org/10.1038/s41422-020-0282-0). 

Reviewer 4: ```suggestion
Previous works already showed that Remdesivir was effective in the treatment of Covid-19 similar viral pathogens, such as SARS-Cov and MERS-Cov, in animal models, while a recent study showed that Remdesivir resulted in 100% protection against the Ebola virus in human liver Cancer cell line, which is also sensitive to the novel Corona virus [@doi:10.1038/s41422-020-0282-0]. 
```

### Treatment of Symptoms

The clinical picture of SARS-CoV-2 infection ranges from being asymptomatic, going through mild symptoms, such as cough, sore throat and fever, to severe pneumonia with respiratory failure which can lead to death. 
The development and the approval of a vaccine against this new microbe will require at least a year. 
Afterwards, the vaccine will face another challenge. 
It will in fact need to be produced in vast quantities in order to be distributed to all countries, while many companies do not have such production capacity. 
While at the moment the most effective way to fight Corona virus is social distancing, scientist are also focusing on other ways to cure the infection. 
Currently, treatments are going in two different directions.
On one side, efforts are focusing on the development of new drugs which can cure the symptoms.
On the other side, scientist are testing new antivirals that can inhibit the virus amplification in human bodies. 
In this section we will describe and analyse the most recent developed drugs that are undergoing new trials, as well as  to face the Corona virus disease. 

#### Tocilizumab

> A recent study carried out on a sample of 191 Covid-19 patients from Wuhan shows that the blood of patients who faced death contained about 74% more interleukin-6 (IL-6) compared to the survivors (doi.org/10.1016/s0140-6736(20)30566-3).

Reviewer 4: ```suggestion
A recent study carried out on a sample of 191 Covid-19 patients from Wuhan shows that the blood of patients who faced death contained about 74% more interleukin-6 (IL-6) compared to the survivors [@doi:ggnxb3].
```

This indicates an inflammatory response out of control or what is called “cytokine storm”.

> In this context IL-6 plays a key role (doi.org/10.1128/mmbr.05015-11).

Reviewer 4: ```suggestion
In this context IL-6 plays a key role [@doi:10.1128/mmbr.05015-11].
```

> IL-6 is a pro-inflammatory cytokine belonging to the family of interleukins, immune system regulators, primarily responsible for immune cell differentiation and activation. Interleukins can regulate both pro- and anti-inflammatory response.

Reviewer 4: ```suggestion
IL-6 is a pro-inflammatory cytokine belonging to the family of interleukins, immune system regulators, primarily responsible for immune cell differentiation and activation.
Interleukins can regulate both pro- and anti-inflammatory response.
```

In this context, severe lung infection reflects the over-production of proinflammatory interleukins. 
In a healthy situation the lung respiratory epithelium together with alveolar macrophages limits the activation of the immune system as well as the alveolar spaces, ensuring homeostasis.
It has been already shown that high level of IL-6, during the infection of other Corona viruses (influenza, SARS COV), lead to the high concentration of other cytokines, such as Tumor Necrosis Factor (TNF) and IL-1𝛽. These cytokines enhance the pro-inflammatory reaction by increase acute-phase signaling, trafficking of immune cells to the site of primary infection, epithelial cell activation, and secondary cytokine production.
The acute phase response to infection results in the heavily damage of the endotelium of blood vessels, which disrupts the balance between pro and anti-inflammatory response [@doi:10.1164/ajrccm.164.10.2104013].

> Thus, the holes generated allow not just for the passage of neutrophils, macrophages and lymphocytes to the site of the infection but also the accumulation of liquids into the lungs, which is the ultimate cause of the death as per Acute Distress Respiratory Syndrome (ADRS) or Severe Acute Respiratory Syndrome (SARS) (doi.org/10.1007/s00281-017-0629-x), also caused by the new coronavirus. 

Reviewer 4: ```suggestion
Thus, the holes generated allow not just for the passage of neutrophils, macrophages and lymphocytes to the site of the infection but also the accumulation of liquids into the lungs, which is the ultimate cause of the death as per Acute Distress Respiratory Syndrome (ADRS) or Severe Acute Respiratory Syndrome (SARS) [@doi:10.1007/s00281-017-0629-x], also caused by the new coronavirus. 
```

Recently Chinese and Italian doctors have found that the the Tocilizumab (actemra, by Roche), a drug commonly used in the rheumatoid arthritis, is very effective in the cure of Covid-19 patients. 

##### Anticipated Mechanism 

Specifically, Tocilizumab is a humanised monoclonal antibody that binds to the receptor of IL-6, de facto inhibiting the IL-6 immune cascade. 

##### Current Evidence

Chinese doctors have started a trial enrolling 188 patients of which 14 with severe lung disease have shown clear sings of improvements, according to their results. 
Also, The AIFA (the Italian Drug Agency) approved the start of a new trial on March 19 recruiting patients at the initial stage of the infection. 
Together with these independent trials, Roche, also in collaboration with the FDA, will start a randomised, double-blind, placebo-controlled phase III trial early April. 
The trial will enroll 330 patients globally, which will be followed for 60 days upon use of the drug via injection to analyse its efficiency/safety (Biopharma-reporter.com).

> However, previous studies on RA showed that in patients treated with TCZ the rate of incident infections in clinical practice patients was higher than the one observed during clinical trial (doi.org/10.1093/rheumatology/ker223). 

Reviewer 4: ```suggestion
However, previous studies on RA showed that in patients treated with TCZ the rate of incident infections in clinical practice patients was higher than the one observed during clinical trial [@doi:10.1093/rheumatology/ker223]. 
```

> Also, RA patients with chronic hepatitis B (HB) infection showed high risk of HB virus reactivation upon TCZ administration in combination with other RA drugs (doi.org/10.1111/1756-185X.13010). 

Reviewer 4: ```suggestion
Also, RA patients with chronic hepatitis B (HB) infection showed high risk of HB virus reactivation upon TCZ administration in combination with other RA drugs [@doi:10.1111/1756-185X.13010]. 
```

Antiviral drugs do not kill the virus, rather they inhibit its amplification by impairing one of these steps. 
Nowadays, many of these drugs act during the biosynthesis step in order to inhibit the virus’ genetic material replication. Importantly, Corona viruses, including Covid-19, are RNA viruses. 
Differently from a DNA virus, which can use the host enzymes to propagate itself, RNA viruses depends on their own polymerase, the RNA-dependent RNA polymerase (RdRP), in order to be replicated [@doi:10.1007/978-1-4939-2438-7; @doi:10.1002/jmv.25761].

#### Nucleotide Analogs

Why one might use nucleotide analogs.

##### Avigan

Avigan (Flavipiravir, from Toyama chemical Fujifilm) is a drug which has been found effective to block viral amplification in the Influenza virus infection. 
Specifically, Avigan is a nucleoside precursor efficiently recognised as guanosine and adenosine analogue by the virus polymerase (RNA-dependent RNA polymerase). 

> While a single incorporation does not influence RNA transcription, multiple events of incorporation lead to the arrest of RNA synthesis (doi.org/10.1371/journal.pone.0068347). Importantly It was already shown that Avigan is able to give 100% coverage against the Ebola virus in mice [@doi:10.1038/s41422-020-0282-0]. 

Reviewer 4: ```suggestion
While a single incorporation does not influence RNA transcription, multiple events of incorporation lead to the arrest of RNA synthesis [@doi:10.1371/journal.pone.0068347].
Importantly It was already shown that Avigan is able to give 100% coverage against the Ebola virus in mice [@doi:10.1038/s41422-020-0282-0]. 
```

This indicates an inflammatory response out of control or what is called “cytokine storm”.
In this context IL-6 plays a key role [@doi:10.1128/mmbr.05015-11].
IL-6 is a pro-inflammatory cytokine belonging to the family of interleukins, immune system regulators, primarily responsible for immune cell differentiation and activation.
Interleukins can regulate both pro- and anti-inflammatory response.
In this context, severe lung infection reflects the over-production of proinflammatory interleukins. 
In a healthy situation the lung respiratory epithelium together with alveolar macrophages limits the activation of the immune system as well as the alveolar spaces, ensuring homeostasis.
It has been already shown that high level of IL-6, during the infection of other Corona viruses (influenza, SARS COV), lead to the high concentration of other cytokines, such as Tumor Necrosis Factor (TNF) and IL-1𝛽. These cytokines enhance the pro-inflammatory reaction by increase acute-phase signaling, trafficking of immune cells to the site of primary infection, epithelial cell activation, and secondary cytokine production.

> The acute phase response to infection results in the heavily damage of the endotelium of blood vessels, which disrupts the balance between pro and anti-inflammatory response [@doi:10.1164/ajrccm.164.10.2104013].

Author: Dear @rando2, the paper I carefully analysed the citokine storm. Here some of the text of the paper:

"As the balance of pro- and anti-inflammatory
mechanisms is critical to maintaining lung immune homeostasis,
it is conceivable that if one or more of these regulatory mechanisms
are absent or aberrantly regulated, then the outcome may
contribute toward a cytokine storm". Then if you go to the paragraph concerning the pathology of the cytokine storm, it says that it causes tissue damage and edema. I guess I tried to summarise and rewrite some of these concept. if these stuff don't look correct to you, you can take them out. I am not an immunologist, but I think those concepts were summarised correctly.

> The acute phase response to infection results in the heavily damage of the endotelium of blood vessels, which disrupts the balance between pro and anti-inflammatory response [@doi:10.1164/ajrccm.164.10.2104013].

Reviewer 2: Hi @aadattoli that makes a lot of sense, I think the DOIs got swapped! (The doi there points to _Cytokine Balance in the Lungs of Patients with Acute Respiratory Distress Syndrome_, but it seems like you're talking about in _Into the Eye of the Cytokine Storm_). I definitely see what you're talking about in that article!

Recently Chinese and Italian doctors have found that the the Tocilizumab (actemra, by Roche), a drug commonly used in the rheumatoid arthritis, may palliate the most severe symptoms associated with COVID-19. 

##### Anticipated Mechanism 

Specifically, Tocilizumab is a humanised monoclonal antibody that binds to the receptor of IL-6, de facto inhibiting the IL-6 immune cascade. 

##### Current Evidence

Chinese doctors have started a trial enrolling 188 patients of which 14 with severe lung disease have shown clear sings of improvements, according to their results. 
Also, The AIFA (the Italian Drug Agency) approved the start of a new trial on March 19 recruiting patients at the initial stage of the infection. 
Together with these independent trials, Roche, also in collaboration with the FDA, will start a randomised, double-blind, placebo-controlled phase III trial early April. 

> The trial will enroll 330 patients globally, which will be followed for 60 days upon use of the drug via injection to analyse its efficiency/safety (Biopharma-reporter.com).

Reviewer 4: Maybe for these upcoming studies, the way to go is perhaps to provide a link at the end of the current evidence section since the data doesn't exist yet?

However, previous studies on RA showed that in patients treated with TCZ the rate of incident infections in clinical practice patients was higher than the one observed during clinical trial [@doi:10.1093/rheumatology/ker223]. 

> Also, RA patients with chronic hepatitis B (HB) infection showed high risk of HB virus reactivation upon TCZ administration in combination with other RA drugs [@doi:10.1111/1756-185X.13010]. 

Reviewer 4: This is really important. If the drug doesn't work but folks survive, this might make them worse off.

These last findings highlight the need to search for a balance between impairing a harmful immune response, such as the one generated by the cytokine storm, and preventing the worsening of the clinical picture of the patients by potential new viral infections. 
This aspect is probably crucial to be investigated further in the trials that are about to start. 

> Perhaps, the TCZ treatment would best suit patients with severely compromised lungs due to the Covid-19 infection and are therefore at greater risk of death, in order to stop the uncontrolled immune response before it’s too late. 

Reviewer 4: I'll probably move this to `Summary` after this merges.

##### Summary

Summarize the state of the symptom management approach.

### Small Molecule Drugs for COVID-19

The replication cycle of a virus within an epithelial host cell includes 6 basic steps which can be shortly summarised as follow: i) attachment of the virus to the host cell; ii) penetration by endocytosis; iii) uncoating, classically defined as the release of viral contents into the host cell; iv) biosynthesis, during which the viral genetic material enters the nucleus where it gets replicated; v) assembly, where viral proteins are translated and new viral particles are assembled; vi) release, when the new viruses are released into the extracellular environment [@isbn:978-1405136457]. 
Antiviral drugs do not kill the virus, rather they inhibit its amplification by impairing one of these steps. 

> Nowadays, many of these drugs act during the biosynthesis step in order to inhibit the virus’ genetic material replication. Importantly, Corona viruses, including Covid-19, are RNA viruses. 

Reviewer 4: ```suggestion
Nowadays, many of these drugs act during the biosynthesis step in order to inhibit the virus’ genetic material replication.
Importantly, SARS-CoV-2 is an RNA virus. 
```

Differently from a DNA virus, which can use the host enzymes to propagate itself, RNA viruses depends on their own polymerase, the RNA-dependent RNA polymerase (RdRP), in order to be replicated [@doi:10.1007/978-1-4939-2438-7; @doi:10.1002/jmv.25761].

#### Nucleotide Analogs

Why one might use nucleotide analogs.

##### Avigan

Avigan (Flavipiravir, from Toyama chemical Fujifilm) is a drug which has been found effective to block viral amplification in the Influenza virus infection. 
Specifically, Avigan is a nucleoside precursor efficiently recognised as guanosine and adenosine analogue by the virus polymerase (RNA-dependent RNA polymerase). 
While a single incorporation does not influence RNA transcription, multiple events of incorporation lead to the arrest of RNA synthesis [@doi:10.1371/journal.pone.0068347].
Importantly It was already shown that Avigan is able to give 100% coverage against the Ebola virus in mice [@doi:10.1038/s41422-020-0282-0]. 
Furthermore, there are evidences that drug might also work against Corona virus infection. 

> For instance, a recent study showed its effectiveness compared to other antivirals such as lopinavir and ritonavir. 

Reviewer 4: Ref?

Author: Cgreene sorry I was working on another review for my lab. I will now take a look at all your suggestions

Author: I also wanted to add more about the mechanism of action of tocilizumab, to be more specific. Will work on it right now

Author: I will add now all the references

Author: This is the ref you were asking about https://www.sciencedirect.com/science/article/pii/S2095809920300631

Author: @cgreene  I accepted all your suggestions and added something more on the mechanism of action of tocilizumab starting from line 40. please let me know whether you think I am going to the right direction with it.

Also lung body scan seem to look better in about 91% of the patients analysed [@doi:10.1016/j.eng.2020.03.007]. 
However, the size of the sample is too small to give a powerful statistics, as well as the choice of the patients did not take into consideration important factors such as previous clinical conditions, sex, while there was no age categorisation. 

###### Current Evidence

A list of current studies and their results, using carefully the information requested in the therapeutic paper tickets.

###### Summary

Summarize the state of the antiviral approach.

##### Remdesivir

Remdesivir (Gilead) is an adenosine analogue which once incorporated into the nascent viral chain causes premature termination of transcription [@doi:10.1038/s41422-020-0282-0].
Remdesivir acts at early stage of infection. 
Previous works already showed that Remdesivir was effective in the treatment of Covid-19 similar viral pathogens, such as SARS-Cov and MERS-Cov, in animal models, while a recent study showed that Remdesivir resulted in 100% protection against the Ebola virus in human liver Cancer cell line, which is also sensitive to the novel Corona virus [@doi:10.1038/s41422-020-0282-0]. 

> Upon FDA approval Gilead enrolled about a 1000 patients from Covid-19 hit countries for two phase III clinical trials to test the efficacy of the drug in adults. 

Reviewer 4: ```suggestion
Upon FDA approval Gilead enrolled about a 1000 patients from COVID-19 hit countries for two phase III clinical trials to test the efficacy of the drug in adults. 
```

> Specifically, one trial will test 400 patients with severe Covid-19 sympotms, while the second trial will randomise the remaining amount of patients with mild symptoms. 

Reviewer 4: ```suggestion
Specifically, one trial will test 400 patients with severe COVID-19 symptoms, while the second trial will randomise the remaining amount of patients with mild symptoms. 
```

### Treatment of Symptoms

The clinical picture of SARS-CoV-2 infection differs dramatically between individuals.
Some are asymptomatic.
Others experience mild COVID-19 symptoms, such as cough, sore throat and fever, while the most severe cases of COVID-19 include severe complications including pneumonia and Acute Respiratory Distress Syndrome (ARDS), which can lead to respiratory failure and death [@url:https://www.cdc.gov/coronavirus/2019-ncov/hcp/clinical-guidance-management-patients.html]. 
Vaccines are one avenue to mitigate harm from viral pathogens, but in the case of a rapidly growing pandemic the longer timeframe of vaccine development and distribution means that there can be a key for treatments that palliate symptoms to avoid the most severe outcomes from infection. 

#### Tocilizumab

> A recent study carried out on a sample of 191 adult COVID-19 in-patients at two Wuhan hospitals found that blood samples taken at admission contained significantly higher concentrations of interleukin-6 (IL-6) in patients who ultimately deceased compared to those who survived; average concentrations of IL-6 remained higher in the deceased group than the surviving group throughout hospitalization [@doi:ggnxb3].

Reviewer 2: ```suggestion
A recent study carried out on a sample of 191 adult COVID-19 in-patients at two Wuhan hospitals found that blood samples taken at admission contained significantly higher concentrations of interleukin-6 (IL-6) in patients who ultimately deceased compared to those who survived; average concentrations of IL-6 remained higher in the deceased group than the surviving group throughout hospitalization [@doi:10/ggnxb3].
```

This suggests that these individuals may be experiencing a "cytokine storm", which refers to an excessive inflammatory response.
IL-6 plays a key role in this response [@doi:10.1128/mmbr.05015-11].
IL-6 is a pro-inflammatory cytokine belonging to the family of interleukins, which are immune system regulators that are primarily responsible for immune cell differentiation. 
Specifically, IL-6 promotes the differentiation of activated B cells into immunoglobulin-producing plasma cells [@pmid:9052836] and acts as a growth factor for hybridoma and myeloma cells [@pmid:8431556; @pmid:2104241]. 
In addition, IL-6 also induces the differentiation of naïve CD4+ T cells into effector T-cell subsets [10.1158/2326-6066.CIR-14-0022].
In this way interleukins regulate both the pro- and anti-inflammatory responses.
In this context, the observation of elevated IL-6 in patients who died may reflect an over-production of proinflammatory interleukins. 

In a healthy situation the lung respiratory epithelium together with alveolar macrophages limits the activation of the immune system, ensuring homeostasis.

> It has been already shown that high level of IL-6, during the infection of other Corona viruses (influenza, SARS COV), lead to the high concentration of other cytokines, such as Tumor Necrosis Factor (TNF) and IL-1𝛽 [doi.org/10.1016/j.virusres.2007.02.007, DOI: 10.1002/path.2067].

Reviewer 2: ```suggestion
The introduction of the S-protein from SARS-CoV to mouse macrophages was found to increase production of IL-6 and TNF-α [@doi:10.1016/j.virusres.2007.02.007], and deceased SARS-CoV patients were found to have intermediate levels of IL-6, IL-1𝛽, and TNF-α expressed in a number of ACE2-expressing cell types sampled from the lung and bronchial tissues during autopsy [@doi:10.1002/path.2067].
However, other reports found the severe respiratory condition ARDS to be associated with elevated concentrations of IL-6 in BALF, but that concentrations of Tumor Necrosis Factor α (TNF-α) and IL-1𝛽 decreased with the onset of ARDS [@doi:10.1164/ajrccm.164.10.2104013].
```

Reviewer 2: Leaving in the BALF result for now because it might be interesting to dig further into this topic!

Vaccines are one avenue to mitigate harm from viral pathogens, but in the case of a rapidly growing pandemic the longer timeframe of vaccine development and distribution means that there can be a key for treatments that palliate symptoms to avoid the most severe outcomes from infection. 
Afterwards, the vaccine will face another challenge. 
Currently, treatments are going in two different directions.
On one side, efforts are focusing on the development of new drugs which can cure the symptoms.
In this section we will describe and analyse the most recent developed drugs that are undergoing new trials, as well as  to face the Corona virus disease. 

#### Tocilizumab

A recent study carried out on a sample of 191 Covid-19 patients from Wuhan shows that the blood of patients who faced death contained about 74% more interleukin-6 (IL-6) compared to the survivors [@doi:ggnxb3].
This suggests that these individuals may be experiencing a "cytokine storm", which refers to an inflammatory response out of control.
In this context IL-6 plays a key role [@doi:10.1128/mmbr.05015-11].

> IL-6 is a pro-inflammatory cytokine belonging to the family of interleukins, immune system regulators, primarily responsible for immune cell differentiation. Specifically, they promote the differentiation of activated B cells into immunoglobulin-producing plasma cells, as well as they represent the growth factor for hybridoma and myeloma cells. In addition, IL-6 also induces differentiation of naïve CD4+ T cells into effector T-cell subsets [10.1158/2326-6066].

Author: This is the reference 10.1158/2326-6066.CIR-14-0022, will immediately change it on the text

> In addition, IL-6 also induces the differentiation of naïve CD4+ T cells into effector T-cell subsets [10.1158/2326-6066.CIR-14-0022].

Reviewer 2: ```suggestion
In addition, IL-6 also induces the differentiation of naïve CD4+ T cells into effector T-cell subsets [@doi:10.1158/2326-6066.CIR-14-0022].
```

In this way interleukins regulate both pro- and anti-inflammatory response.
In this context, the observation of elevated IL-6 in patients who died may reflect an over-production of proinflammatory interleukins. 
In a healthy situation the lung respiratory epithelium together with alveolar macrophages limits the activation of the immune system as well as the alveolar spaces, ensuring homeostasis.

> It has been already shown that high level of IL-6, during the infection of other Corona viruses (influenza, SARS COV), lead to the high concentration of other cytokines, such as Tumor Necrosis Factor (TNF) and IL-1𝛽.

Author: This is one of the papers: 
Up-regulation of IL-6 and TNF-α induced by SARS-coronavirus spike protein in murine macrophages via NF-κB pathway [doi.org/10.1016/j.virusres.2007.02.007]
Here another reference:
Expression of elevated levels of pro‐inflammatory cytokines in SARS‐CoV‐infected ACE2+ cells in SARS patients: relation to the acute lung injury and pathogenesis of SARS [doi.org/10.1002/path.2067]. I thought I already added them yesterday, couldn't you see them? Do these ref look fine to you? I think these results are important in the context tocilizumab use, because IL-1Beta is crucial to the citokine storm reaction: "IL-1beta is a key cytokine driving proinflammatory activity in bronchoalveolar lavage fluid of patients with lung injury", from the paper: into the eye of the citokine storm. What do you think?

> It has been already shown that high level of IL-6, during the infection of other Corona viruses (influenza, SARS COV), lead to the high concentration of other cytokines, such as Tumor Necrosis Factor (TNF) and IL-1𝛽.

Reviewer 2: I wasn't seeing them before, but I'm updating it now!

These cytokines enhance the pro-inflammatory reaction by increase acute-phase signaling, trafficking of immune cells to the site of primary infection, epithelial cell activation, and secondary cytokine production.
The acute phase response to infection results in the heavily damage of the endotelium of blood vessels, which disrupts the balance between pro and anti-inflammatory response [@doi:10.1164/ajrccm.164.10.2104013].
Thus, the holes generated allow not just for the passage of neutrophils, macrophages and lymphocytes to the site of the infection but also the accumulation of liquids into the lungs, which is the ultimate cause of the death as per Acute Distress Respiratory Syndrome (ADRS) or Severe Acute Respiratory Syndrome (SARS) [@doi:10.1007/s00281-017-0629-x], also caused by the new coronavirus. 
Recently Chinese and Italian doctors have found that the the Tocilizumab (actemra, by Roche), a drug commonly used in the rheumatoid arthritis, may palliate the most severe symptoms associated with COVID-19. 

##### Anticipated Mechanism 

> Human IL-6 is a glycoprotein of 26 kDa and it consists of 184 amino acids containing 2 potential N-glycosylation sites and four cysteine residues. IL-6 binds to its receptor either in the insoluble (IL-6R) and soluble (sIL-6R) form [10.1158/2326-6066]. 

Reviewer 2: ```suggestion
Human IL-6 is a glycoprotein of 26 kDa and it consists of 184 amino acids containing 2 potential N-glycosylation sites and four cysteine residues. IL-6 binds to its receptor either in the insoluble (IL-6R) and soluble (sIL-6R) form [@doi:10.1158/2326-6066]. 
```

> Human IL-6 is a glycoprotein of 26 kDa and it consists of 184 amino acids containing 2 potential N-glycosylation sites and four cysteine residues. IL-6 binds to its receptor either in the insoluble (IL-6R) and soluble (sIL-6R) form [@doi:10.1158/2326-6066]. 

Reviewer 2: ```suggestion
Human IL-6 is a glycoprotein of 26 kDa and it consists of 184 amino acids containing 2 potential N-glycosylation sites and four cysteine residues. IL-6 binds to its receptor either in the insoluble (IL-6R) and soluble (sIL-6R) form.
```

> The receptor specificity determines the type of signaling. Specifically, the binding of IL-6 to the cell membrane receptor IL-6R gives rise to the "classical transduction of the signaling", while to binding to sIL-6R generate the so called "trans-signaling" [doi: 10.1016/j.cytogfr.2012.04.001, DOI: 10.1042/bj3000281]. 

Reviewer 2: ```suggestion
The receptor specificity determines the type of signaling. Specifically, the binding of IL-6 to the cell membrane receptor IL-6R gives rise to the "classical transduction of the signaling", while to binding to sIL-6R generate the so called "trans-signaling" [@doi:10.1016/j.cytogfr.2012.04.001; @doi:10.1042/bj3000281]. 
```

> IL-6 signaling occurs through 3 independent pathways: the Janus-activated kinase (JAK)-STAT3 pathway, the Ras/Mitogen-Activated Protein Kinases (MAPK) pathway and the Phosphoinositol-3 Kinase (PI3K)/Akt pathway [doi.org/10.3389/fimmu.2016.00604]. 

Reviewer 2: ```suggestion
IL-6 signaling occurs through 3 independent pathways: the Janus-activated kinase (JAK)-STAT3 pathway, the Ras/Mitogen-Activated Protein Kinases (MAPK) pathway and the Phosphoinositol-3 Kinase (PI3K)/Akt pathway [@doi:10.3389/fimmu.2016.00604]. 
```

> The ultimate result of the IL-6 cascade is to direct transcriptional activity of various promoters of pro-inflammatory cytokines, such as IL-1 and TFN, including IL-6 own regulation through the activity of NF-κB [doi.org/10.3389/fimmu.2016.00604].

Reviewer 2: ```suggestion
The ultimate result of the IL-6 cascade is to direct transcriptional activity of various promoters of pro-inflammatory cytokines, such as IL-1 and TFN, including IL-6 own regulation through the activity of NF-κB [@doi:10.3389/fimmu.2016.00604].
```

> Particularly, IL-6 synthesis is thightly regulated both transcriptionally and post-trascriptionally. In this context, it has been shown that viral proteins can enhance transcription of the IL-6 gene, via strenghtening the DNA-binding activity between several trasciptional factors and IL-6 gene-cis-regulatory elements [doi.org/10.2492/inflammregen.33.054]. 

Reviewer 2: ```suggestion
Particularly, IL-6 synthesis is tightly regulated both transcriptionally and post-transcriptionally . In this context, it has been shown that viral proteins can enhance transcription of the IL-6 gene, via strengthening the DNA-binding activity between several transciptional factors and IL-6 gene-cis-regulatory elements [@doi:10.2492/inflammregen.33.054]. 
```

Tocilizumab is a humanised monoclonal antibody that binds both to the insoluble and soluble receptor of IL-6, de facto inhibiting the IL-6 immune cascade. 

##### Current Evidence

> Chinese doctors have started a trial enrolling 188 patients of which 14 with severe lung disease have shown clear sings of improvements, according to their results [https://www.genengnews.com/virology/coronavirus/genentech-launches-phase-iii-trial-of-actemra-as-coronavirus-treatment/]. 

Reviewer 2: ```suggestion
Chinese doctors have started a trial enrolling 188 patients of which 14 with severe lung disease have shown clear sings of improvements, according to their results [@url:https://www.genengnews.com/virology/coronavirus/genentech-launches-phase-iii-trial-of-actemra-as-coronavirus-treatment/]. 
```

> Also, The AIFA (the Italian Drug Agency) approved the start of a new trial on March 19 recruiting patients at the initial stage of the infection [https://www.univadis.co.uk/viewarticle/covid-19-italy-launches-an-independent-trial-on-tocilizumab-715741]. 

Reviewer 2: ```suggestion
Also, The AIFA (the Italian Drug Agency) approved the start of a new trial on March 19 recruiting patients at the initial stage of the infection [@url:https://www.univadis.co.uk/viewarticle/covid-19-italy-launches-an-independent-trial-on-tocilizumab-715741]. 
```

Antiviral drugs do not kill the virus, rather they inhibit its amplification by impairing one of these steps. 
Nowadays, many of these drugs act during the biosynthesis step in order to inhibit the virus’ genetic material replication.
Importantly, SARS-CoV-2 is an RNA virus. 
Differently from a DNA virus, which can use the host enzymes to propagate itself, RNA viruses depends on their own polymerase, the RNA-dependent RNA polymerase (RdRP), in order to be replicated [@doi:10.1007/978-1-4939-2438-7; @doi:10.1002/jmv.25761].

#### Nucleotide Analogs

Why one might use nucleotide analogs.

##### Avigan

Avigan (Flavipiravir, from Toyama chemical Fujifilm) is a drug which has been found effective to block viral amplification in the Influenza virus infection. 
Specifically, Avigan is a nucleoside precursor efficiently recognised as guanosine and adenosine analogue by the virus polymerase (RNA-dependent RNA polymerase). 
While a single incorporation does not influence RNA transcription, multiple events of incorporation lead to the arrest of RNA synthesis [@doi:10.1371/journal.pone.0068347].

> Importantly It was already shown that Avigan is able to give 100% coverage against the Ebola virus in mice [@doi:10.1038/s41422-020-0282-0]. 

Reviewer 2: ```suggestion
Importantly It was already shown that Avigan is able to give 100% coverage against the Ebola virus in mice [@doi:10.1016/j.antiviral.2014.02.014]. 
```

Furthermore, there are evidences that drug might also work against Corona virus infection. 

> For instance, a recent study showed its effectiveness compared to other antivirals such as lopinavir and ritonavir. 

Reviewer 2: ```suggestion
For instance, a recent study showed its effectiveness compared to other antivirals such as lopinavir and ritonavir [@doi:10.1016/j.eng.2020.03.007]. 
```

It has been already shown that high level of IL-6, during the infection of other Corona viruses (influenza, SARS COV), lead to the high concentration of other cytokines (TFN, IL-1𝛽) with the results of heavily damaging the endotelium of blood vessels and disrupting the balance between pro and anti-inflammatory response (Park et al. 2001).
The holes generated allow not just for the passage of neutrophils, macrophages and lymphocytes to the site of the infection but also the accumulation of liquids into the lungs, which is the ultimate cause of the death as per Acute Distress Respiratory Syndrome (ADRS) or Severe Acute Respiratory Syndrome (SARS) (Channappanavar and Perlman 2017), also caused by the new Corona virus . 

> Recently Chinese and Italian doctors have found that the the Tocilizumab (acterma, by Roche), a drug commonly used in the rheumatoid arthritis, is very effective in the cure of Covid-19 patients. Specifically, Tocilizumab is a humanised monoclonal antibody that binds to the receptor of IL-6, de facto inhibiting the IL-6 immune cascade. Chinese doctors have started a trial enrolling 188 patients o which 14 with severe lung disease have shown clear sings of improvements, according to their results. Also, The AIFA (the Italian Drug Agency) approved the start of a new trial on March 19 recruiting patients at the initial stage of the infection. Together with these independent trials, Roche, also in collaboration with the FDA, will start a randomised, double-blind, placebo-controlled phase III trial early April. The trial will enroll 330 patients globally, which will be followed for 60 days upon use of the drug via injection to analyse its efficiency/safety (Biopharma-reporter.com).

Reviewer 5: I think the Tocilizumab trial is exciting, but it may be worth mentioning both the pros and potential risks of using an immunosuppressive agent in the treatment of an infection. Typically, an immunosuppressive drug would be avoided in a person with a known infection because shutting down the immune system might allow the infection to spread. For example, this paper demonstrates the higher risk of infections in tocilizumab users with rheumatoid arthritis: https://academic.oup.com/rheumatology/article/51/5/852/1804905  Other papers also show risk of re-activation of Hep B in rheumatoid arthritis patients on tocilizumab: https://onlinelibrary.wiley.com/doi/abs/10.1111/1756-185X.13010

In this case, however, the severe "cytokine storm" in response to the virus is deleterious in and of itself, hence the possible benefit of using an IL-6 blocker in the short-term to curtail the effects of the virus. As a result, there's a delicate balance between blocking a harmful immune cascade (potentially allowing other infections to take over) and preventing worsening of the viral infection. I think a brief discussion of the risk-benefit analysis (why the trial might work, and what its potential drawbacks are) may be warranted. Perhaps this might mean that the treatment is suitable only when patients are at greatest risk of fatality ("severe lung disease" patients used in the study), and the immediate effects of the cytokine storm on worsening lung disease must be stopped before it's too late. Of course, these last thoughts are just my speculations; we can't know until the trial results are out.

Author: Dear dianerafi, I integrated your suggestions starting at line 38. Please ket me know your feedback as soon as you get a chance.
Best regards

Reviewer 5: Looks good to me. I'll update if anything else comes to mind. I found the safety indications for TCZ here https://www.actemra.com/safety.html in case it's of interest (i.e. that the drug is usually contraindicated in patients with infection). It would be interesting if there is any evidence for TCZ being used to fight an infection rather than an autoimmune response, though I haven't been able to find that so far.

Antivirals

Background: The replication cycle of a virus within an epithelial host cell includes 6 basic steps which can be shortly summarised as follow: i) attachment of the virus to the host cell; ii) penetration by endocytosis; iii) uncoating, classically defined as the release of viral contents into the host cell; iv) biosynthesis, during which the viral genetic material enters the nucleus where it gets replicated; v) assembly, where viral proteins are translated and new viral particles are assembled; vi) release, when the new viruses are released into the extracellular environment (ISBN-13: 978-1405136457). 

> Antiviral drugs do not kill the virus, rather they inhibit its amplification by impairing one of these steps. Nowadays, many of these drugs act during the biosynthesis step in order to inhibit the virus’ genetic material replication. Importantly, Corona viruses, including Covid-19, are RNA viruses. Differently from a DNA virus, which can use the host enzymes to propagate itself, RNA viruses depends on their own polymerase, the RNA-dependent RNA polymerase (RdRP), in order to be replicated (doi.org/10.1007/978-1-4939-2438-7; doi.org/10.1002/jmv.25761). 

Reviewer 6: I would suggest deleting the "in order" in the last sentence so you end up with "the RNA-dependent RNA polymerase to be replicated"

Summarize the state of the neutralizing antibody approach.

#### Interferons

Interferons (IFNs) are a family of cytokines crucial to activate the first (innate) immune system response against viral infections. 

> Interferons are classified into three categories based on their receptor specificity: type I, II and III (doi.org/10.1128/mmbr.05015-11). 

Reviewer 4: ```suggestion
Interferons are classified into three categories based on their receptor specificity: type I, II and III [@doi:10.1128/mmbr.05015-11]. 
```

> Specifically, IFNs I (IFN-𝛼 and 𝛽) and II (IFN-𝛾 ) induce the expression of antiviral proteins which bring the viral RNA to degradation (doi.org/10.1016/j.jcv.2003.11.013). 

Reviewer 4: ```suggestion
Specifically, IFNs I (IFN-𝛼 and 𝛽) and II (IFN-𝛾 ) induce the expression of antiviral proteins which bring the viral RNA to degradation [@doi:10.1016/j.jcv.2003.11.013]. 
```

> Among these IFNs, IFN- 𝛽 was already found to strongly inhibit the replication of other corona viruses, such as SARS-Cov, in cell culture, while IFN-𝛼 and 𝛾 were shown to be less effective in this context (doi.org/10.1016/j.jcv.2003.11.013). 

Reviewer 4: ```suggestion
Among these IFNs, IFN- 𝛽 was already found to strongly inhibit the replication of other corona viruses, such as SARS-Cov, in cell culture, while IFN-𝛼 and 𝛾 were shown to be less effective in this context [@doi:10.1016/j.jcv.2003.11.013]. 
```

There are evidences that patients with higher susceptibility to develop Acute respiratory distress syndrome (ARDS) show indeed deficiency of IFN- 𝛽. 

> For instance, upon other Corona viruses infection IFN- 𝛽 expression and synthesis is impaired, so that the virus can in fact escape the innate immune response (doi.org/10.1016/j.virusres.2014.07.024). 

Reviewer 4: ```suggestion
For instance, upon other Corona viruses infection IFN- 𝛽 expression and synthesis is impaired, so that the virus can in fact escape the innate immune response [@doi:10.1016/j.virusres.2014.07.024]. 
```
